src/modules.d.ts

```diff
@@ -33,13 +33,13 @@ declare namespace Electron {
         fromProcessAndFrameIds: (processId: number, frameId: number) => WebContents;
         getOwnerBrowserWindow: () => BrowserWindow | void;
         mainFrameRoutingId: number;
-        session: Session;
+        readonly session: Session;
```

Comment:
Let's delete this line and use the definition from electron.d.ts (just built locally and that's fine)

Reply:
done